manifests/init.pp

```diff
@@ -72,6 +73,7 @@
   Hash $config                           = { 'server' => 'https://acme-v02.api.letsencrypt.org/directory' },
   String $cron_scripts_path              = "${facts['puppet_vardir']}/letsencrypt",
   String $cron_owner_group               = 'root',
+  String $root_file_owner_group          = 'root',
```

Comment:
I think this should be correct for the supported OSes. So it should be set to wheel on the right BSDs. In that case it should not be in `init.pp` at all, but only in data. See `configure_epel`.

The alternative is to not use a name but rather ID 0. That is common between unices.

Reply:
I've set this in the data now, thanks

Reply:
since empty strins aren't allowed, please update to:
```suggestion
  String[1] $root_file_owner_group       = 'root',
```